main.nf

```diff
@@ -1033,35 +1162,35 @@ process VARSCAN2_BCFTOOLS {
     !params.skip_variants && 'varscan2' in callers
 
     input:
-    tuple val(sample), val(single_end), path(bam), path(vcf) from ch_sort_bam_varscan2_bcftools.join(ch_varscan2_highfreq_consensus, by: [0,1])
+    tuple val(sample), val(single_end), path(bam), path(vcf) from ch_markdup_bam_varscan2_consensus.join(ch_varscan2_highfreq_consensus, by: [0,1])
     path fasta from ch_fasta
 
     output:
-    tuple val(sample), val(single_end), path("*consensus.masked.fa") into ch_bcftools_consensus_masked
-    tuple val(sample), val(single_end), path("*consensus.fa")
+    tuple val(sample), val(single_end), path("*consensus.masked.fa") into ch_varscan2_consensus
+    path "*consensus.fa"
 
     script:
+    prefix = "${sample}.AF${params.max_allele_freq}"
     """
-    cat $fasta | bcftools consensus ${vcf[0]} > ${sample}.consensus.fa
+    cat $fasta | bcftools consensus ${vcf[0]} > ${prefix}.consensus.fa
 
     bedtools genomecov \\
         -bga \\
         -ibam ${bam[0]} \\
         -g $fasta \\
-        | awk '\$4 < 20' | bedtools merge > ${sample}.mask.bed
+        | awk '\$4 < 20' | bedtools merge > ${prefix}.mask.bed
```

Comment:
If we use 10 as threshold depth in variant calling we shuld use here 10 also, and this should be the same general parameter as above.

Reply:
Fixed https://github.com/nf-core/viralrecon/pull/93/commits/a495c1c40bd21085c21247ad812e51bcbb90088f